### DISCUSSION TITLE
Dependency updates

### DISCUSSION THREAD

**Discussion by ghost** [pulsar-edit/.github#34] [General]
I feel like dependencies need updating. A lot of dependencies need updating, and a chunk of them are security updates.

We should update security ones as a priority, because they are for security, and they are important.

We may as well need to update dependencies that don't have security, if any of the team members want to.

This is probably a step of modernization.

**Comment by confused-Techie** [comment]
Totally agree there, and it is something in progress. Although the bigger things are getting updated like Electron, and Node, which hopefully will then allow us to make other updates that don't totally break things. Since iirc we are updating from Node and Electron versions that are several years old.

But maybe are there any specific deps that have critical security updates that should be first priorities after getting these other ones going?
